Author:      Mike Sperber <sperber@deinprogramm.de>
Description: In cmuscheme48.el, use `make-temp-file' instead of constant file name.
Index: scheme48-1.9/emacs/cmuscheme48.el
===================================================================
--- scheme48-1.9.orig/emacs/cmuscheme48.el	2012-12-27 17:43:30.000000000 +0100
+++ scheme48-1.9/emacs/cmuscheme48.el	2014-05-27 19:48:09.000000000 +0200
@@ -58,7 +58,7 @@
    (let ((end (point)))
      (beginning-of-defun)
      (if losep
-	 (let ((loser "/tmp/s48lose.tmp"))
+         (let ((loser (make-temp-file "s48lose")))
 	   (write-region (point) end loser)
 	   (scheme48-load-file loser))
 	 (scheme48-send-region (point) end)))))
